### PR TITLE
Upgrade to eslint@2 and friends.

### PR DESCRIPTION
**Pull request by MoOx**
Closes #295

### CODE REVIEW

File: docs/content/docs/advanced/good-practices.md [hunk 1]
```diff
@@ -24,16 +24,18 @@ To remove the default config, you can replace the ``eslintConfig`` in your
 ```json
 {
   "eslintConfig": {
-    "parser": "babel-eslint",
```
Comment (thangngoc89) [inline docs/content/docs/advanced/good-practices.md]: Why delete babel-eslint ?

File: docs/content/docs/advanced/good-practices.md [hunk 2]
```diff
@@ -24,16 +24,18 @@ To remove the default config, you can replace the ``eslintConfig`` in your
...
     "extends": "eslint:recommended",
-    "ecmaFeatures": {
-      "modules": true,
-      "experimentalObjectRestSpread": true
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "experimentalObjectRestSpread": true,
```
Comment (thangngoc89) [inline docs/content/docs/advanced/good-practices.md]: Default boilerplate is using package.json to set eslint config. JSON doesn't allow trailing comma

File: docs/content/docs/advanced/good-practices.md [hunk 3]
```diff
@@ -24,16 +24,18 @@ To remove the default config, you can replace the ``eslintConfig`` in your
...
+      },
     },
     "env": {
       "node": true,
       "browser": true,
-      "es6": true
+      "es6": true,
```
Comment (thangngoc89) [inline docs/content/docs/advanced/good-practices.md]: Trailing comma

**Comment by thangngoc89** [comment]
You forgot to sync eslint config to boilerplate template.

File: docs/content/docs/advanced/good-practices.md (hunk 1 above)
Comment (MoOx) [inline docs/content/docs/advanced/good-practices.md]: mistake

File: docs/content/docs/advanced/good-practices.md (hunk 2 above)
Comment (MoOx) [inline docs/content/docs/advanced/good-practices.md]: > Default boilerplate is using package.json to set eslint config.

Yeah, so what is the problem? This place is for people that want minimal eslint config (not the shareable one)

File: docs/content/docs/advanced/good-practices.md (hunk 3 above)
Comment (MoOx) [inline docs/content/docs/advanced/good-practices.md]: oups

**Comment by MoOx** [comment]
>  You forgot to sync eslint config to boilerplate template.

Not sure what is the issue.

**Comment by thangngoc89** [comment]
Oops. I thought the eslintConfig in boilerplate isn't compatiable with eslint 2. looks like it is

**Pull request merged**